insights/debian/lintian-overrides

```diff
@@ -1,2 +1,3 @@
 # Legitimate name/pseudonym used in upstream copyright, not a typo
 ubuntu-insights: spelling-error-in-copyright Unknwon Unknown
+libinsights-dev: spelling-error-in-copyright Unknwon Unknown
```

Comment:
Can’t we use * or something like this? My overrides is rusty, but I think we can then avoid the per-deb files too that way.